enterprise/tomcat5/src/org/netbeans/modules/tomcat5/deploy/TomcatManagerImpl.java

```diff
@@ -530,7 +530,7 @@ public synchronized void run () {
                 // Set up an authorization header with our credentials
                 TomcatProperties tp = tm.getTomcatProperties();
                 String input = tp.getUsername () + ":" + tp.getPassword ();
-                String auth = new String(Base64.encode(input.getBytes()));
+                String auth = new String(Base64.getEncoder().encode(input.getBytes()));
```

Comment:
Please don't use the String constructor (especially not without a charset), but use `encodeToString`

Reply:
Done as suggested. Thanks